mdata/cache/accnt/flat_accnt.go

```diff
@@ -139,12 +161,19 @@ func (a *FlatAccnt) eventLoop() {
 						Ts:     payload.ts,
 					},
 				)
+			case evnt_del_met:
+				payload := event.pl.(*DelMetPayload)
+				a.delMet(payload.metric)
+			case evnt_get_total:
+				payload := event.pl.(*GetTotalPayload)
+				a.getTotal(payload.res_chan)
 			case evnt_stop:
 				return
 			case evnt_reset:
 				a.metrics = make(map[string]*FlatAccntMet)
 				a.total = 0
 				a.lru.reset()
+				cacheSizeUsed.SetUint64(a.total)
```

Comment:
it seems throughout all code, cacheSizeUsed is kept in sync with a.total ; so why do we track the same information in two different numbers? can't we just answer reads by looking at cacheSizeUsed?

Reply:
right, that makes sense... actually then we could also get rid of the entire method to get the current total, because we can simply read `cacheSizeUsed.Peek()` which uses an atomic read operation